Matrix-math example - ImportError...

Following the example of matrix-math [fd5b49c](https://github.com/bytecodealliance/componentize-py/commit/fd5b49cd1e7303dab8a4c93c0432aa07f790b04e) I get the following error when running `componentize-py -d ../../wit -w matrix-math componentize app -o matrix-math.wasm`:
```

warning: site-packages directory not found under /home/jes/gitrepositories/wasmcloud_test/my-py-component-wasmtime16-0-0/myenv/lib
Traceback (most recent call last):
  File "/home/jes/.local/bin/componentize-py", line 8, in <module>
    sys.exit(script())
             ^^^^^^^^
AssertionError: Traceback (most recent call last):
  File "/0/app.py", line 5, in <module>
    import numpy
  File "/0/numpy/__init__.py", line 157, in <module>
    from . import random
  File "/0/numpy/random/__init__.py", line 180, in <module>
    from . import _pickle
  File "/0/numpy/random/_pickle.py", line 1, in <module>
    from .mtrand import RandomState
ImportError: dynamic module does not define module export function (PyInit_mtrand)


Caused by:
    ImportError: dynamic module does not define module export function (PyInit_mtrand)
```

I am using python 3.12 and have created an enviroment using virtualenv
I am more experienced in Rust than python and have been unable to figure out if it is an error on my side or with the example.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "componentize-py"
-version = "0.9.0"
+version = "0.9.1"
 edition = "2021"
 exclude = ["cpython"]
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,9 +17,9 @@ zstd = "0.13.0"
 componentize-py-shared = { path = "shared" }
 wasmparser = "0.118.0"
 wasm-encoder = "0.38.1"
-# TODO: switch to release once https://github.com/bytecodealliance/wasm-tools/pull/1317 is released
-wit-parser = { git = "https://github.com/bytecodealliance/wasm-tools", rev = "231fa81b" }
-wit-component = { git = "https://github.com/bytecodealliance/wasm-tools", rev = "231fa81b" }
+# TODO: switch to release once https://github.com/bytecodealliance/wasm-tools/pull/1317 and https://github.com/bytecodealliance/wasm-tools/pull/1341 are released
+wit-parser = { git = "https://github.com/bytecodealliance/wasm-tools", rev = "928a946c" }
+wit-component = { git = "https://github.com/bytecodealliance/wasm-tools", rev = "928a946c" }
 indexmap = "2.1.0"
 bincode = "1.3.3"
 heck = "0.4.1"

--- a/examples/cli/README.md
+++ b/examples/cli/README.md
@@ -13,7 +13,7 @@ snapshot, which may differ from later revisions.
 ## Prerequisites
 
 * `Wasmtime` 16.0.0 (later versions may use a different, incompatible `wasi-cli` snapshot)
-* `componentize-py` 0.9.0
+* `componentize-py` 0.9.1
 
 Below, we use [Rust](https://rustup.rs/)'s `cargo` to install `Wasmtime`.  If
 you don't have `cargo`, you can download and install from
@@ -21,7 +21,7 @@ https://github.com/bytecodealliance/wasmtime/releases/tag/v16.0.0.
 
 ```
 cargo install --version 16.0.0 wasmtime-cli
-pip install componentize-py==0.9.0
+pip install componentize-py==0.9.1
 ```
 
 ## Running the demo

--- a/examples/http/README.md
+++ b/examples/http/README.md
@@ -14,7 +14,7 @@ which may differ from later revisions.
 ## Prerequisites
 
 * `Wasmtime` 16.0.0 (later versions may use a different, incompatible `wasi-http` snapshot)
-* `componentize-py` 0.9.0
+* `componentize-py` 0.9.1
 
 Below, we use [Rust](https://rustup.rs/)'s `cargo` to install `Wasmtime`.  If
 you don't have `cargo`, you can download and install from
@@ -22,7 +22,7 @@ https://github.com/bytecodealliance/wasmtime/releases/tag/v16.0.0.
 
 ```
 cargo install --version 16.0.0 wasmtime-cli
-pip install componentize-py==0.9.0
+pip install componentize-py==0.9.1
 ```
 
 ## Running the demo

--- a/examples/matrix-math/README.md
+++ b/examples/matrix-math/README.md
@@ -11,7 +11,7 @@ within a guest component.
 ## Prerequisites
 
 * `wasmtime` 16.0.0 (later versions may use a different, incompatible `wasi-cli` snapshot)
-* `componentize-py` 0.9.0
+* `componentize-py` 0.9.1
 * `NumPy`, built for WASI
 
 Note that we use an unofficial build of NumPy since the upstream project does
@@ -23,7 +23,7 @@ https://github.com/bytecodealliance/wasmtime/releases/tag/v16.0.0.
 
 ```
 cargo install --version 16.0.0 wasmtime-cli
-pip install componentize-py==0.9.0
+    pip install componentize-py==0.9.1
 curl -OL https://github.com/dicej/wasi-wheels/releases/download/v0.0.1/numpy-wasi.tar.gz
 tar xf numpy-wasi.tar.gz
 ```

--- a/examples/tcp/README.md
+++ b/examples/tcp/README.md
@@ -14,7 +14,7 @@ snapshot, which may differ from later revisions.
 ## Prerequisites
 
 * `Wasmtime` 16.0.0 (later versions may use a different, incompatible `wasi-cli` snapshot)
-* `componentize-py` 0.9.0
+* `componentize-py` 0.9.1
 
 Below, we use [Rust](https://rustup.rs/)'s `cargo` to install `Wasmtime`.  If
 you don't have `cargo`, you can download and install from
@@ -22,7 +22,7 @@ https://github.com/bytecodealliance/wasmtime/releases/tag/v16.0.0.
 
 ```
 cargo install --version 16.0.0 wasmtime-cli
-pip install componentize-py==0.9.0
+pip install componentize-py==0.9.1
 ```
 
 ## Running the demo

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -7,7 +7,7 @@ features = ["pyo3/extension-module"]
 
 [project]
 name = "componentize-py"
-version = "0.9.0"
+version = "0.9.1"
 description = "Tool to package Python applications as WebAssembly components"
 readme = "README.md"
 license = { file = "LICENSE" }

